serializers/src/main/java/io/pravega/schemaregistry/codec/Codecs.java

```diff
@@ -53,6 +53,13 @@ public CodecType getCodecType() {
 
         @Override
         public void encode(ByteBuffer data, ByteArrayOutputStream bos) {
+            if (data.hasArray()) {
```

Comment:
This pattern is repeated in a few places, it may be worth making a utility function.

Reply:
created issue https://github.com/pravega/schema-registry/issues/70